andela/suyabay
# Fix login javascript issues

#### What does this PR do?
-  Fix user login problem 
#### Description of Task to be completed?
- Add a class selector to the submit button
- Add an event(click) listener to the button
#### How should this be manually tested?
- Visit `https://suyabay-staging.herokuapp.com/login` and enter your login details, if successfully you should be logged in into Suyabay dashboard.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?

121038179
#### Screenshots (if appropriate)
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
